distrib/docker/entrypoint_netatalk.sh

```diff
@@ -288,7 +289,8 @@ else
     case "$TESTSUITE" in
         spectest)
             set -x
-            afp_spectest $TEST_FLAGS -"$AFP_VERSION" -h "$AFP_HOST" -p "$AFP_PORT" -u "$AFP_USER" -d "$AFP_USER2" -w "$AFP_PASS" -s "$SHARE_NAME" -S "$SHARE_NAME2"
+            afp_spectest $TEST_FLAGS -"$AFP_VERSION" -h "$AFP_HOST" -p "$AFP_PORT" -u "$AFP_USER" -d "$AFP_USER2" -w "$AFP_PASS" -s "$SHARE_NAME" -S "$SHARE_NAME2" || true
```

Comment:
How do you propose that we bubble up test failures to the CI system with this change? Maybe capture the exit code in a variable and return with it after posting the server logs?

Reply:
Ahh, good catch! This was needed while debugging some crashes.
I have just updated the code to capture the return code and then send it after optionally printing the server logs - this should allow us to support automated testing and human debugging.
The new server log printing has proven to be invaluable! The ability to see both client and server spectest logs at the same time with test filtering was so useful!